torch/nn/intrinsic/qat/modules/conv_fused.py

```diff
@@ -79,15 +79,27 @@ def freeze_bn_stats(self):
         self.bn.training = False
         return self
 
-    def _forward(self, input):
-        running_std = torch.sqrt(self.bn.running_var + self.bn.eps)
-        scale_factor = self.bn.weight / running_std
-        scaled_weight = self.weight_fake_quant(self.weight * scale_factor.reshape([-1, 1, 1, 1]))
+    # Runs the conv forward with scaled weights and unscales the weights.
+    #
+    # Note: this is implemented as a standalone function to match the
+    #   rules of graph mode quantization, which will mimic this code.
+    #   In particular, the inputs and outputs of this function should be
+    #   observed, and no intermediate values of this function should be
+    #   observed.
+    def _conv_forward_and_unscale(self, input, scaled_weight, scale_factor):
```

Comment:
I actually mean an independent function. this is a method for the class

Reply:
rule for graph mode right now is we will go through CallMethod, but we will not go through the CallFunction in the list

Reply:
yep, we're on the same page.  Not intending to directly reuse this code, more structuring the eager mode code the same way we aligned for the graph mode IR, for easier testing during development.

let me mark this PR as wip, not ready for review yet.